data/graphics.s

```diff
@@ -619,7 +619,7 @@ gTextPal:
 
     .global gCharSet
 gCharSet:  @ Charset
-    .incbin "baserom.gba", 0x001D312C, 0x2A800
+    .incbin "graphics/fonts/charset.4bpp.lz"
```

Comment:
i didn't have time to review this yesterday but this should not be lz compressed it's just tiles